<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Spectrogram</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pathEdit" />
      </item>
      <item>
       <widget class="QPushButton" name="browseButton" >
        <property name="text" >
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Pixels per second</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="ppsSpin" >
        <property name="maximum" >
         <number>10000</number>
        </property>
        <property name="singleStep" >
         <number>10</number>
        </property>
        <property name="value" >
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea" >
      <property name="alignment" >
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>1</y>
         <width>513</width>
         <height>290</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <widget class="CursorLabel" name="imageLabel" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>[image]</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progress" >
      <property name="value" >
       <number>0</number>
      </property>
      <property name="textVisible" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" >
      <item>
       <widget class="QCheckBox" name="commandCheck" >
        <property name="text" >
         <string>Command</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="commandEdit" >
        <property name="text" >
         <string>xmms -p</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="centeredCheck" >
        <property name="text" >
         <string>Centered</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="playButton" >
        <property name="palette" >
         <palette>
          <active>
           <colorrole role="Button" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>170</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Button" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>170</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Button" >
            <brush brushstyle="SolidPattern" >
             <color alpha="255" >
              <red>170</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="text" >
         <string>Play</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton" >
        <property name="text" >
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CursorLabel</class>
   <extends>QLabel</extends>
   <header>cursorlabel</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
